js/scripts.js

```diff
@@ -0,0 +1,24 @@
+function Button(text) {
+	this.text = text || 'Hello';
+}
+
+Button.prototype = {
+	create: function() {
+		this.$element = $('<button>');
+	}
+}
+
+Button.prototype = {
+	create: function() {
+		var self = this;
+		this.$element = $('<button>');
+		this.$element.text(this.text);
+		this.$element.click(function() {
+			alert(self.text);
+		$('body').append(this.$element);
```

Comment:
you should append body outside of this function, in this case it will be added to body only when somebody clicks on it ;p but you can't click before you add it to body